keyboards/dotmod/dymium65/readme.md

```diff
@@ -0,0 +1,13 @@
+# Peaker Hotswap PCB by FJLaboratories
```

Comment:
This file's contents need to conform to the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md?plain=1).